client/web/compose/src/components/PageBlocks/RecordConfigurator.vue

```diff
@@ -42,11 +117,55 @@ export default {
 
   components: {
     FieldPicker,
+    VueSelect,
   },
 
   extends: base,
+
+  computed: {
+    documentationURL () {
+      // eslint-disable-next-line no-undef
+      const [year, month] = VERSION.split('.')
+      return `https://docs.cortezaproject.org/corteza-docs/${year}.${month}/integrator-guide/compose-configuration/index.html`
+    },
+
+    fieldOptions () {
+      const nonRequiredFields = this.module.fields.filter(f => !f.isRequired)
+      const unConditionedFields = nonRequiredFields.filter(f => !this.options.fieldConditions.find(fc => fc.field.name === f.name))
+      const fields = unConditionedFields.map(f => {
+        return {
+          name: f.name,
+          label: f.label,
+          fieldID: f.fieldID,
+          isRequired: f.isRequired,
+        }
+      })
+      return fields
+    },
+  },
+
+  created () {
+    console.log(this.fieldOptions)
+  },
+
+  methods: {
+    add () {
+      if (this.options.fieldConditions.length >= this.module.fields.length) return
+      this.options.fieldConditions.push({
+        field: {},
+        condition: '',
+      })
+      this.$root.$emit('checkState', { items: this.options.fieldConditions, indicator: 'field' })
+    },
+
+    deleteCondition (i) {
+      this.options.fieldConditions.splice(i, 1)
+      this.$root.$emit('checkState', { items: this.options.fieldConditions, indicator: 'field' })
+    },
+  },
 }
 </script>
+
 <style lang="scss" scoped>
```

Comment:
Delete this whole style section, not sure why its still here (I know you didn't add it but its not needed)